packs/duo.yml

```diff
@@ -0,0 +1,10 @@
+AnalysisType: pack
+PackID: PantherManaged.Duo
+Description: Group of all Duo detections
+DisplayName: Panther Duo Pack
+PackDefinition:
+  IDs:
+    - Duo.User.Action.Fraudulent
+    - Duo.User.Denied.AnomalousPush
+    # Globals used in these detections
+    - panther_base_helpers
```

Comment:
does this belong here?

Reply:
So far as I can tell, yes it does since it'll allow importing of the helpers used in the rule files.  All the other packs have it too

Reply:
👍 the filenames without the `.py` extension for any global_helpers used in the detections should indeed be in the pack.yml file.